metron-platform/metron-parsers/src/main/java/org/apache/metron/parsers/bolt/ParserBolt.java

```diff
@@ -181,7 +185,8 @@ public void declareOutputFields(OutputFieldsDeclarer declarer) {
   @Override
   public void updateConfig(String path, byte[] data) throws IOException {
     super.updateConfig(path, data);
-    if (path.startsWith(ConfigurationType.PARSER.getZookeeperRoot() + "/" + getSensorType())) {
+    String pathWithoutTrailingSlash = path.replaceAll("/+$", "");
+    if (pathWithoutTrailingSlash.equals(ConfigurationType.PARSER.getZookeeperRoot() + "/" + getSensorType())) {
```

Comment:
I'm sorry I don't follow.  ConfigurationType.PARSER.getZookeeperRoot() is constant and sensorType is a leaf node in Zookeeper.  Maybe an example will help.